src/afterfact.rs

```diff
@@ -319,13 +323,15 @@ fn emit_csv<W: std::io::Write>(
             .unwrap_or(&0) as usize
     };
     let mut author_list_cache: HashMap<CompactString, Nested<String>> = HashMap::new();
+    let mut prev_message: HashMap<CompactString, Profile> = HashMap::new();
     for (message_idx, time) in MESSAGEKEYS
         .lock()
         .unwrap()
         .iter()
         .sorted_unstable()
         .enumerate()
     {
+        // remove duplicate dataのための前レコード分の情報を保持する変数
```

Comment:
Could you move the comment as well?

Reply:
@fukusuket Thanks for your comment.
I adjusted comment in https://github.com/Yamato-Security/hayabusa/pull/1104/commits/e28d221e2d93a3cfad09f888f0a279c7a15af41e . 
I am very sorry for the repeated requests.
Please could you confirm this?